ftw/file/JSONDetails.py

```diff
@@ -0,0 +1,24 @@
+from Acquisition import aq_inner
+from ftw.file.content.file import File
+from Products.CMFCore.utils import getToolByName
+from Products.TinyMCE.adapters.JSONDetails import JSONDetails
+import simplejson as json
+
+
+class FtwJSONDetails(JSONDetails):
+
+    def getDetails(self):
+        details_json = super(FtwJSONDetails, self).getDetails()
+        details = json.loads(details_json)
+
+        utility = getToolByName(aq_inner(self.context), 'portal_tinymce')
+        image_portal_types = utility.imageobjects.splitlines()
```

Comment:
``image_portal_types`` is unused, therefore ``utility`` is unused; can be removed, I think.